serialization/README.md

```diff
@@ -2,13 +2,36 @@
 
 ## Serialization info
 
-_TBD_
+This directory contains the specifications and examples for all supported SPDX formats.
```

Comment:
Suggest this be slightly tweaked to say "This directory contains the specifications and examples for all supported SPDX serialization formats."

Reply:
@sbarnum : you can use the `suggestion` feature to make your suggestion easier to read. It is then rendered as:

```suggestion
This directory contains the specifications and examples for all supported SPDX serialization formats.
```

Reply:
Updated per suggestion